static/js/formatters-internal.js

```diff
@@ -522,6 +523,28 @@ export function price(fieldValue = {}, locale) {
   return price.toLocaleString(localeForFormatting, { style: 'currency', currency: currencyCode });
 }
 
+/**
+ * Returns a localized price range string for the given price range ($-$$$$) and country code (ISO format)
+ * @param {Object} pricerange The price range from LiveAPI entity
+ * @param {string} countrycode The country code from LiveAPI entity (e.g. profile.address.countryCode)
+ * @return {string} The price range with correct currency symbol formatting according to country code
+ */
+export function priceRange(priceRange, countryCode) {
```

Comment:
Minor nit, but I find it confusing to have a function called `priceRange` that takes in a `priceRange`. How about naming this something like `localizePriceRange`?

Reply:
I think keeping the function name as priceRange would be more consistent with the other formatter functions like Formatter.price(). I can rename priceRange param to `defaultPriceRange` or `entityPriceRange`?

Reply:
Sure, `defaultPriceRange` or `priceRangeFieldValue` sgtm!